DoctorWho.DB/Validation/EpisodeForCreateDtoValidator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using DoctorWho.DB.Resources;
+using FluentValidation;
+
+namespace DoctorWho.DB.Validation
+{
+    public class EpisodeForCreateDtoValidator:AbstractValidator<EpisodeForCreateDto>
+    {
+        public EpisodeForCreateDtoValidator()
+        {
+            RuleFor(e => e.AuthorId).NotEmpty();
+            RuleFor(e => e.DoctorId).NotEmpty();
+            RuleFor(e => e.SeriesNumber.ToString())
+                   .Must(e => e.Length == 5);
```

Comment:
in the requirement ,length should be 10 ,, but I defined series number as int32 ,, so lenght of 10 is exceeded the range .
maybee I can change series number type to string in entity class and then add new migration , should I try it ?